rfcs/text/0012-orchestrator-field-set.md

```diff
@@ -33,15 +92,57 @@ Stage 3: Add or update all remaining field definitions. The list should now be e
 
 ## Usage
 
-<!--
-Stage 1: Describe at a high-level how these field changes will be used in practice. Real world examples are encouraged. The goal here is to understand how people would leverage these fields to gain insights or solve problems. ~1-3 paragraphs.
--->
+The `orchestrator` field set will be used to capture typical concepts employed
+by container orchestrators to manage resources. The key intent of this is to create
+a consistent method by which audit logs from container orchestrators can
+be compared. For example, this would allow the creation of open source detection
+rulesets for suspicious Kubernetes events based on audit logs, which can be easily
+transferred from one cluster to another without depending on the specifics of
+parsing implementations.
+
+This might also have use in performance and monitoring tooling which exists around
+container orchestrators, allowing for the definition of shareable dashboards and
+alert definitions.
 
 ## Source data
 
-<!--
-Stage 1: Provide a high-level description of example sources of data. This does not yet need to be a concrete example of a source document, but instead can simply describe a potential source (e.g. nginx access log). This will ultimately be fleshed out to include literal source examples in a future stage. The goal here is to identify practical sources for these fields in the real world. ~1-3 sentences or unordered list.
--->
+Examples of source data include:
+
+- [Kubernetes audit logs][1]
+- [Kubernetes node logs][2]
+- [HashiCorp Nomad audit logs][3]
+- [Falco alert logs][4]
+
+### Kubernetes audit log
+
+```json
+{
+  "_index": "filebeat-7.7.0-2020.12.31-000001",
+  "_type": "_doc",
+  "_id": "KbmPuXYBaTdcl42uyGfl",
+  "_version": 1,
+  "_score": null,
+  "_source": {
+    "@timestamp": "2020-12-31T16:09:35.735Z",
+    "log": {
+      "offset": 7248566,
+      "file": {
+        "path": "/tmp/host-logs/kube-apiserver-audit.log"
+      }
+    },
+    "cloud.provider": "gcp",
+    "event.action": "create",
+    "orchestrator.cluster": "test-dev",
+    "orchestrator.type": "Kubernetes",
+    "orchestrator.subresource": "attach",
+    "orchestrator.resource.type": "pods",
+    "orchestrator.namespace": "default",
+    "orchestrator.resource.name": "test",
+    "orchestrator.api_version": "v1",
+    "user.name": "system:serviceaccount:test"
+  }
+}
+```
```

Comment:
Could you also include some example for other orchestrator?

Reply:
Yep - I've added an example for Hashicorp Nomad.

Reply:
Thanks.